src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -25,10 +26,13 @@ import { equalGlyphSelection } from "./scene-controller.js";
 import {
   registerVisualizationLayerDefinition,
   strokeRoundNode,
+  strokeSquareNode,
 } from "./visualization-layer-definitions.js";
 import { copyComponent } from "/core/var-glyph.js";
 
-const handleMarginValue = 10;
+const transformationHandleMargin = 6;
+const transformationHandleSize = 8;
+const rotationHandleSizeFactor = 1.5;
```

Comment:
```suggestion
const rotationHandleSizeFactor = 1.2;
```
For me, this feels more balanced between resize and rotate.